server/utils/similarity.js

```diff
@@ -19,6 +21,19 @@ class Similarity {
     return similarityMethod === 0 ? playlistObject.cosineSimilarityScore : playlistObject.ownSimilarityScore
   }
 
+  updateSimilarityScores = async (allPlaylists) => {
```

Comment:
To make this even better. Make the method 'static' to the utility and move allPlaylists into the method so it doesn't need any arguments. Helpful article:
https://stackoverflow.com/questions/40987138/how-create-static-functions-objects-in-javascript-nodejs-es6

What you want is to be able to just call something like `Similarity.updateSimilarityScores()`.